eng/scripts/TypeSpec-Requirement.ps1

```diff
@@ -44,23 +43,37 @@ else {
     # ToDo: Fetch main and query local git repo to prevent issues with rate limiting
```

Comment:
Is this todo now obsolete?

Reply:
I'm willing to say "yes" and delete this, until we see a real-world failure.